readthedocs/settings/base.py

```diff
@@ -311,24 +299,21 @@ def INSTALLED_APPS(self):  # noqa
             apps.append("readthedocsext.cdn")
             apps.append("readthedocsext.donate")
             apps.append("readthedocsext.spamfighting")
-        if self.RTD_EXT_THEME_ENABLED:
-            apps.append("readthedocsext.theme")
         if self.SHOW_DEBUG_TOOLBAR:
             apps.append("debug_toolbar")
 
+        if ext_theme:
+            apps.append("readthedocsext.theme")
```

Comment:
Should probably always include this, as without exttheme the app would no longer work

Reply:
I tried that, but I had to add this conditional because there are some scenarios where exttheme is not present: embed testings, for example.

We can work on polishing that in another PR if we want; but I didn't want to block this work due to that.